### DISCUSSION TITLE
[Suggestions] Adventure map, Foundry, adjust the position of the Master Swordsman sprite after upgrade in the Status Window

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#4605] [Ideas]
When a troop of Swordsmen has been upgraded to a troop of Master Swordsmen, the position of the creature's sprite in the Status Window is switched.
This is the same in the original game.

In fheroes2, I think that this position should be adjusted so that there is not this effect of position shift.
The sprites of the other creatures keep the same position before and after being upgraded.

In fheroes2:

![Foundry sprite position 01](https://user-images.githubusercontent.com/43583994/141120242-aab2adca-7efe-4516-a225-bd76082f4ba5.png)

![Foundry sprite position 02](https://user-images.githubusercontent.com/43583994/141120254-14ebdfcd-df49-44f6-bce0-b19aa176a5c9.png)

Open the two images in different tabs in your browser, enlarge the images and switch between the two tabs alternately.

![Foundry sprite position recadré 01](https://user-images.githubusercontent.com/43583994/141120318-20975cf8-ed95-473f-b106-60c141e5ce85.png) ![Foundry sprite position recadré 02](https://user-images.githubusercontent.com/43583994/141120321-f090afc9-b071-4d3f-9f6b-58d1645a0676.png)

Here is the save file:
[Before Foundry 01.zip](https://github.com/ihhub/fheroes2/files/7513013/Before.Foundry.01.zip)

**Comment by zenseii** [comment]
Just thought I'd add as I'm sure you have noticed, but this is due to the Master Swordsman having that pony tail on the helmet which shifts the rest of the body some pixels to the right.
